ci/encrypt_gbq.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+
+GBQ_JSON_FILE=$1
+GBQ_PROJECT_ID=$2
+
+if [[ $GBQ_JSON_FILE != *.json ]]; then
+    echo "ERROR: Expected *.json file"
+    exit 1
+fi
+
+if [[ ! -f $GBQ_JSON_FILE ]]; then
+    echo "ERROR: File $GBQ_JSON_FILE does not exist"
+    exit 1
+fi
+
+if [[ ! -n "$GBQ_PROJECT_ID" ]]; then
+    echo "ERROR: Please specify the Google BigQuery Project Id"
+    exit 1
+fi
+
+echo "Adding your project $GBQ_PROJECT_ID to .travis.yml ..."
+sed -i -e "s/.*export\ GBQ_PROJECT_ID.*/\ \ \ \ \ \ \ \ export\ "\
+"GBQ_PROJECT_ID='$GBQ_PROJECT_ID';/g" ../.travis.yml
+
+echo "Encrypting $GBQ_JSON_FILE..."
+read -d "\n" TRAVIS_KEY TRAVIS_IV <<<$(travis encrypt-file $GBQ_JSON_FILE \
+travis_gbq.json.enc -f | grep -o "\w*_iv\|\w*_key");
+
+echo "Adding your secure key to .travis.yml ..."
```

Comment:
It would be better to move this check OUT of ``.travis.yml`` and actually put in a separate (yaml or txt) type of file, then you don't have to do all of this crazy sub/replace stuff, just have a file like

```
TRAVIS_IV=....
GBQ_PROJECT_ID=...
```

then simply re-write the file

then .travis.yml section is:

 ```
ci\travis_process_gbq_encryption.sh
```

ci\travis_process_gbq_encryption.sh
```
# read the variables / project id from the file
 - if [ -n '$encrypted_f44557bb53ea_iv' ]; then
         openssl aes-256-cbc -K $encrypted_f44557bb53ea_key
         -iv $encrypted_f44557bb53ea_iv -in ci/travis_gbq.json.enc
          -out ci/travis_gbq.json -d;
         export GBQ_PROJECT_ID='pandas-140401';
```